src/Clapi/NamespaceTracker.hs

```diff
@@ -92,7 +92,7 @@ nstProtocol_ = forever $ liftedWaitThen fwd rev
         >> (lift $ sendRev $ Right $ ServerDisconnect i) >> handleDisconnect i
 
 nonClaim :: TrpDigest -> Bool
-nonClaim (TrpDigest _ _ dd cops _) = dd == alEmpty && null cops
+nonClaim (TrpDigest _ _ _ dd cops _) = dd == alEmpty && null cops
```

Comment:
There are quite a few of these data structures that feel to me like we're moving towards wanting the projection functions so that we don't have to keep track of exactly how many arguments we're ignoring! E.g.
```haskell
nonClaim trpd = trdpDataDigest dd == alEmpy && null (trpdContainerOps dd)
```